ts/javascript/project.test.ts

```diff
@@ -0,0 +1,31 @@
+import { describe, it } from "https://deno.land/std@0.206.0/testing/bdd.ts";
+import * as garn from "./mod.ts";
+import { getStorePath, nixpkgsInput } from "../testUtils.ts";
+import { assert } from "https://deno.land/std@0.206.0/assert/mod.ts";
+import { renderFlakeFile } from "../nix.ts";
+import { formatFlake } from "../internal/runner.ts";
+import * as path from "https://deno.land/std/fs/copy.ts";
+import { existsSync } from "https://deno.land/std@0.201.0/fs/exists.ts";
+
+describe("mkNpmProject", () => {
+  it("produces correct store path structure", () => {
+    const tempDir = Deno.makeTempDirSync({ prefix: "garn-test" });
+    path.copySync("./examples/npm-project/", `${tempDir}/npm-project`);
```

Comment:
I think overall it'd be nicer to not use the `examples` folder in here. Could we just write out all the needed files from the test? Maybe just a tiny example with one flat dependency?